src/components/Api.js

```diff
@@ -0,0 +1,104 @@
+const url = table => `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${table}`;
+
+const fetchAndSortTodos = async (table, sortOptions = undefined) => {
```

Comment:
actually every variable always defaults to `undefined` until we put something into it, so you can remove this default and get the same results =D

```suggestion
const fetchAndSortTodos = async (table, sortOptions) => {
```